### PR TITLE
Feature/lattice 1269 entity type entity set collections

### PR DESCRIPTION
**Pull request by katherinebernstein**

### CODE REVIEW

**Review by makosblade — APPROVED** [review]

File: src/main/resources/rhizome.yaml [hunk 1]
```diff
@@ -14,15 +14,6 @@ hazelcast:
     seed-nodes: 
         - "localhost"
 
-hazelcast-clients:
```
Comment (you-leee) [inline src/main/resources/rhizome.yaml]: Is this intentional or because of merge?

Comment (katherinebernstein) [reply src/main/resources/rhizome.yaml]: definitely because of merge :) thanks!

Comment (you-leee) [reply src/main/resources/rhizome.yaml]: np :)

**Review by you-leee — APPROVED** [review]

**Review by pckeyes — APPROVED** [review]
Looks good to me!

**Review by geekbeast — APPROVED** [review]

**Pull request merged**